projects/github-actions/repo-gardening/src/utils/parse-content/find-priority.js

```diff
@@ -2,37 +2,78 @@ const debug = require( '../debug' );
  */
 function findPriority( body ) {
+	let priority = 'TBD';
+
+	debug( `find-priority: Looking for priority indicators in issue body: ${ body }` );
+
 	// Look for priority indicators in body.
 	const priorityRegex =
-		/###\sImpact\n\n(?<impact>.*)\n\n###\sAvailable\sworkarounds\?\n\n(?<blocking>.*)\n/gm;
+		/###\sSite\sowner\simpact\n\n(?<impact>.*)\n\n###\sSeverity\n\n(?<severity>.*)\n\n###\sWhat\sother\simpact\(s\)\sdoes\sthis\sissue\shave\?\n\n(?<extra>.*)\n/gm;
 	let match;
 	while ( ( match = priorityRegex.exec( body ) ) ) {
-		const [ , impact = '', blocking = '' ] = match;
+		const { impact = '', extra = '' } = match.groups || {};
+		let { severity = '' } = match.groups || {};
 
 		debug(
-			`find-priority: Reported priority indicators for issue: "${ impact }" / "${ blocking }"`
+			`find-priority: Reported priority indicators for issue: "${ impact }" / "${ severity }" / "${ extra }"`
 		);
 
-		if ( blocking === 'No and the platform is unusable' ) {
-			return impact === 'One' ? 'High' : 'BLOCKER';
-		} else if ( blocking === 'No but the platform is still usable' ) {
-			return 'High';
-		} else if ( blocking === 'Yes, difficult to implement' ) {
-			return impact === 'All' ? 'High' : 'Normal';
-		} else if ( blocking !== '' && blocking !== '_No response_' ) {
-			return impact === 'All' || impact === 'Most (> 50%)' ? 'Normal' : 'Low';
+		// Folks can provide additional information that can bump severity.
+		// We also do not want that extra information to downgrade the severity.
+		if ( extra !== '' && extra !== '_No response_' && extra !== 'No revenue impact' ) {
+			if (
+				( extra === 'Individual site owner revenue' || extra === 'Agency or developer revenue' ) &&
+				severity !== 'Critical'
+			) {
+				severity = 'Major';
+			} else if ( extra === 'Platform revenue' ) {
+				severity = 'Critical';
+			}
+		}
+
+		const impactIndicators = {
+			isolated: 'Less than 20% of all',
+			scattered: 'Between 20% and 60% of all',
+			widespread: 'More than 60% of all',
+		};
+
+		if ( severity === 'Critical' ) {
+			priority = impact === impactIndicators.isolated ? 'High' : 'BLOCKER';
+		} else if ( severity === 'Major' ) {
+			if ( impact === impactIndicators.widespread ) {
+				priority = 'BLOCKER';
+			} else if ( impact === impactIndicators.scattered ) {
+				priority = 'High';
+			} else {
+				priority = 'Normal';
+			}
+		} else if ( severity === 'Moderate' ) {
+			if ( impact === impactIndicators.widespread ) {
+				priority = 'High';
+			} else if ( impact === impactIndicators.scattered ) {
+				priority = 'Normal';
+			} else {
+				priority = 'Low';
+			}
+		} else if ( severity !== '' && severity !== '_No response_' ) {
+			priority = impact === impactIndicators.widespread ? 'Normal' : 'Low';
+		} else {
+			priority = 'TBD';
 		}
-		return 'TBD';
 	}
 
-	debug( `find-priority: No priority indicators found.` );
-	return 'TBD';
+	debug(
+		`find-priority: ${
+			priority === 'TBD' ? 'No ' : ' '
```

Comment:
If `priority` got changed, this will produce
```
find-priority:  priority indicators found. Priority set to Whatever
```
Should there be something other than just a space in the else case?